.travis.yml

```diff
@@ -4,7 +4,6 @@ python:
   - "3.7"
   - "3.6"
   - "3.5"
-  - "2.7"
```

Comment:
Should remove 3.5 also, that was not a good one :)

Reply:
:+1: Removed also 3.5 support and checks